src/sentry/replays/migrations/0001_init_replays.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.28 on 2022-07-19 17:51
+
+import django.utils.timezone
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not. Generally
+    # we don't want to run in a transaction here, since for long running operations like data
+    # back-fills this results in us locking an increasing number of rows until we finally commit.
+    atomic = False
+
+    initial = True
+
+    dependencies = []
+
+    operations = [
+        migrations.CreateModel(
+            name="ReplayRecordingSegment",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("project_id", sentry.db.models.fields.bounded.BoundedBigIntegerField()),
```

Comment:
Now that you're in the same database as project do you want to have a foreign key?

You should also consider adding ReplayRecordingSegment to the [list of children for Project in the deletions task](https://github.com/getsentry/sentry/blob/8be757c0a7d0394e297cf099a9a1c8a3fc7edc65/src/sentry/deletions/defaults/project.py#L17).

Reply:
added to the deletions task. if we decide to split this out into its own database, would project_id being a foreign key make that any more difficult?

Reply:
Are we putting this in the same database as project? I thought we were going to put it on the `files` db?

Reply:
I don't think it matters which database its on to start while we're in early alpha / beta, but yeah i think a separate database long term is the right move. i'm not sure if `files`, or `attachments` or a new database entirely would be the right home for it. in any case we can leave this as not-a-foreign-key

Reply:
If you do put these models on the `files` db, you'll also need to update `getsentry upgrade` to run migrations for the `files` connection in `conf/settings/prod.py`. Currently migrations do not run on the `files` primaries during deploys.